webpack.config.development.js

```diff
@@ -3,71 +3,113 @@
  * Build config for development process that uses Hot-Module-Replacement
  * https://webpack.github.io/docs/hot-module-replacement-with-webpack.html
```

Comment:
Remove the link to webpack v1 doc or replace with v2 doc
https://webpack.js.org/concepts/hot-module-replacement/